{"name": "test_login[chromium]", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.TimeoutError: Locator.click: Timeout 30000ms exceeded.\nCall log:\nwaiting for get_by_text(\"Войти\")", "trace": "header = <components.header.Header object at 0x000001CF719CFF50>\nlogin = <components.login.Login object at 0x000001CF74EF5460>\n\n    def test_login (header,login):\n        header.visit()\n>       login.click_login_button()\n\ntest_login.py:5: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\components\\login.py:11: in click_login_button\n    self.page.get_by_text(\"Войти\").click()\nD:\\Program Files\\Python\\Python312\\Lib\\site-packages\\playwright\\sync_api\\_generated.py:14929: in click\n    self._sync(\nD:\\Program Files\\Python\\Python312\\Lib\\site-packages\\playwright\\_impl\\_locator.py:156: in click\n    return await self._frame.click(self._selector, strict=True, **params)\nD:\\Program Files\\Python\\Python312\\Lib\\site-packages\\playwright\\_impl\\_frame.py:488: in click\n    await self._channel.send(\"click\", locals_to_params(locals()))\nD:\\Program Files\\Python\\Python312\\Lib\\site-packages\\playwright\\_impl\\_connection.py:59: in send\n    return await self._connection.wrap_api_call(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <playwright._impl._connection.Connection object at 0x000001CF74EF7920>\ncb = <function Channel.send.<locals>.<lambda> at 0x000001CF752C4F40>\nis_internal = False\n\n    async def wrap_api_call(\n        self, cb: Callable[[], Any], is_internal: bool = False\n    ) -> Any:\n        if self._api_zone.get():\n            return await cb()\n        task = asyncio.current_task(self._loop)\n        st: List[inspect.FrameInfo] = getattr(task, \"__pw_stack__\", inspect.stack())\n        parsed_st = _extract_stack_trace_information_from_stack(st, is_internal)\n        self._api_zone.set(parsed_st)\n        try:\n            return await cb()\n        except Exception as error:\n>           raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\nE           playwright._impl._errors.TimeoutError: Locator.click: Timeout 30000ms exceeded.\nE           Call log:\nE           waiting for get_by_text(\"Войти\")\n\nD:\\Program Files\\Python\\Python312\\Lib\\site-packages\\playwright\\_impl\\_connection.py:514: TimeoutError"}, "steps": [{"name": "Открываем Хэдер на главной странице", "status": "passed", "start": 1732175265534, "stop": 1732175265694}, {"name": "Нажимаем кнопку 'Войти' в хедере", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.TimeoutError: Locator.click: Timeout 30000ms exceeded.\nCall log:\nwaiting for get_by_text(\"Войти\")\n\n", "trace": "  File \"D:\\Program Files\\Python\\Python312\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\volod\\PycharmProjects\\python_studies\\RRS_Python_2024_Fall\\Python_project\\lesson5\\components\\login.py\", line 11, in click_login_button\n    self.page.get_by_text(\"Войти\").click()\n  File \"D:\\Program Files\\Python\\Python312\\Lib\\site-packages\\playwright\\sync_api\\_generated.py\", line 14929, in click\n    self._sync(\n  File \"D:\\Program Files\\Python\\Python312\\Lib\\site-packages\\playwright\\_impl\\_sync_base.py\", line 115, in _sync\n    return task.result()\n           ^^^^^^^^^^^^^\n  File \"D:\\Program Files\\Python\\Python312\\Lib\\site-packages\\playwright\\_impl\\_locator.py\", line 156, in click\n    return await self._frame.click(self._selector, strict=True, **params)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\Program Files\\Python\\Python312\\Lib\\site-packages\\playwright\\_impl\\_frame.py\", line 488, in click\n    await self._channel.send(\"click\", locals_to_params(locals()))\n  File \"D:\\Program Files\\Python\\Python312\\Lib\\site-packages\\playwright\\_impl\\_connection.py\", line 59, in send\n    return await self._connection.wrap_api_call(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\Program Files\\Python\\Python312\\Lib\\site-packages\\playwright\\_impl\\_connection.py\", line 514, in wrap_api_call\n    raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\n"}, "start": 1732175265695, "stop": 1732175295709}], "parameters": [{"name": "browser_name", "value": "'chromium'"}], "start": 1732175265534, "stop": 1732175295710, "uuid": "6c97d89b-d10c-42a7-80c3-79395403934c", "historyId": "40feb9a76f809b218c9d56c1d9a7ba42", "testCaseId": "a707b9555172d26c5c1ec2d556e674f3", "fullName": "tests.test_login#test_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "matusha"}, {"name": "thread", "value": "17032-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}